### PR TITLE
Add DAKEFPVF722X8

### PR DESCRIPTION
**Pull request by ThanYangFPV**
Schematics and configuration files have been emailed to [cloudtargets@betaflight.com](mailto:cloudtargets@betaflight.com).

Please review. Thank you.

### CODE REVIEW

File: configs/DAKEFPVF722X8/config.h [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+#define USE_GYRO_SPI_MPU6500
+#define USE_GYRO_SPI_ICM42688P
+
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#define USE_CAMERA_CONTROL
```
Comment (haslinghuis) [inline configs/DAKEFPVF722X8/config.h:44]:
```suggestion
#ifndef USE_CAMERA_CONTROL
#define USE_CAMERA_CONTROL
#endif
```

File: configs/DAKEFPVF722X8/config.h [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    TIMER_PIN_MAP( 0, PC6 , 2, 0) \
+    TIMER_PIN_MAP( 1, PC7 , 2, 1) \
+    TIMER_PIN_MAP( 2, PC8 , 2, 1) \
+    TIMER_PIN_MAP( 3, PC9 , 2, 0) \
+    TIMER_PIN_MAP( 4, PA8 , 1, 0) \
+    TIMER_PIN_MAP( 5, PA15, 1, 0) \
+    TIMER_PIN_MAP( 6, PB7 , 1, 0) \
+    TIMER_PIN_MAP( 7, PA1 , 1, 0) \
+    TIMER_PIN_MAP( 8, PA0 , 2, 0) \
+    TIMER_PIN_MAP( 9, PB0 , 2, 0)
```
Comment (haslinghuis) [inline configs/DAKEFPVF722X8/config.h:100]:
```suggestion
    TIMER_PIN_MAP( 9, PB0 , 2, -1)
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/DAKEFPVF722X8/config.h [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        DAKEFPVF722X8
+#define MANUFACTURER_ID   DAKE
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_MPU6500
```
Comment (ot0tot) [inline configs/DAKEFPVF722X8/config.h:31]: You should not be using the 6500 in new designs 
```suggestion

```

File: configs/DAKEFPVF722X8/config.h [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6500
```
Comment (ot0tot) [inline configs/DAKEFPVF722X8/config.h:35]:
```suggestion

```

File: configs/DAKEFPVF722X8/config.h [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+#define UART5_TX_PIN         PC12
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART4_RX_PIN         PC11
+#define UART5_RX_PIN         PD2
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PC14
+#define LED1_PIN             PC15
```
Comment (ot0tot) [inline configs/DAKEFPVF722X8/config.h:71]: These pins should not be used to drive LEDs

File: configs/DAKEFPVF722X8/config.h [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+    TIMER_PIN_MAP( 0, PC6 , 2, 0) \
+    TIMER_PIN_MAP( 1, PC7 , 2, 1) \
+    TIMER_PIN_MAP( 2, PC8 , 2, 1) \
+    TIMER_PIN_MAP( 3, PC9 , 2, 0) \
+    TIMER_PIN_MAP( 4, PA8 , 1, 0) \
+    TIMER_PIN_MAP( 5, PA15, 1, 0) \
+    TIMER_PIN_MAP( 6, PB7 , 1, 0) \
+    TIMER_PIN_MAP( 7, PA1 , 1, 0) \
+    TIMER_PIN_MAP( 8, PA0 , 2, 0) \
+    TIMER_PIN_MAP( 9, PB0 , 2, -1)
```
Comment (ot0tot) [inline configs/DAKEFPVF722X8/config.h:102]: Motors split between too many timers. M1-4 on TIM8, but M5-8 are on TIM1, TIM2, and TIM4.

**Review by ot0tot — APPROVED** [review]

File: configs/DAKEFPVF722X8/config.h [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+
+#define ADC2_DMA_OPT        1
+
+#define ADC_INSTANCE ADC2
+#define FLASH_SPI_INSTANCE SPI3
+#define MAX7456_SPI_INSTANCE SPI2
+
+#define BARO_I2C_INSTANCE (I2CDEV_1)
```
Comment (haslinghuis) [inline configs/DAKEFPVF722X8/config.h:106]:
```suggestion
#define BARO_I2C_INSTANCE I2CDEV_1
```

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**